assets/state-vgpu-manager/0210_clusterrole.yaml

```diff
@@ -23,14 +23,15 @@ rules:
   - ""
   resources:
   - pods
-  - pods/eviction
   verbs:
-  - create
-  - delete
   - get
   - list
-  - patch
-  - update
+- apiGroups:
+    - ""
+  resources:
+    - pods/eviction
```

Comment:
let's indent the yaml list items in the same level as the list.

Reply:
Done.